etna/transforms/math/limit.py

```diff
@@ -0,0 +1,122 @@
+from typing import List
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.datasets import set_columns_wide
+from etna.transforms.base import ReversibleTransform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders"""
```

Comment:
It is better to add formulas for bounding method.(forward and backward) [see](https://github.com/etna-team/etna/blob/1d20ffc07f90e76cc9fb0717a6e9887551ebd442/etna/models/seasonal_ma.py#L18) + make sure that it renders ok in docs(see link for current docs version in PR)

You can also add link to the bounding method